radixdlt/src/main/java/com/radixdlt/network/addressbook/PeerManager.java

```diff
@@ -15,26 +15,25 @@
  * language governing permissions and limitations under the License.
  */
 
-package org.radix.network2.addressbook;
+package com.radixdlt.network.addressbook;

```

Comment:
Apart from package move, minor tidy ups here to remove Interfaces class, and tidy Sonar tech debt.